assets/app/features/upgrade/i18n/en.json

```diff
@@ -32,50 +32,50 @@
                 },
                 "mode": {
                     "title": "Upgrade Mode",
-                    "description": "After all mandatory checks are successful the upgrade mode will be shown here...",
+                    "description": "After all mandatory checks are successful the upgrade mode of your cloud will be shown here...",
                     "normal": {
-                        "title": "Your cloud is now prepared to begin the upgrade in normal mode.",
+                        "title": "Your cloud is now prepared to begin the upgrade process in normal mode.",
                         "button": "Proceed in Normal Mode",
-                        "description1": "In normal mode, each node will be rebooted during the upgrade process, including the workloads running in them.",
-                        "description2": "In order to proceed in non-disruptive mode your cloud needs to have the pacemaker barclamp correctly configured and applied amd at least one free node as large or larger than any other node in your clouds compute plane.",
+                        "description-title": "IMPORTANT!",
+                        "description1": "In normal mode, the workloads running in each node will be live-migrated to another node. Note that you will lose connection to your VMs when upgrading the control node(s).",
+                        "description2": "In order to proceed in non-disruptive mode your cloud needs to be Highly Available and have enough free compute resources. See the deployment guide for details.",
                         "description3": "Be sure to re-run the checks if you make any changes to your cloud after the initial run."
                     },
                     "nondisruptive": {
                         "title": "Your cloud is now prepared to begin the upgrade process in non-disruptive mode.",
-                        "description1": "In non-disruptive mode all workloads should continue running but we strongly recommend picking your nose and wiggeling your finger at it for a while first.",
+                        "description1": "In non-disruptive mode all workloads should continue running and remain connected.",
```

Comment:
I wouldn't go for _will_ here as there's always some possibility of problems with workloads during migration. I don't like "remain connected" too. Sounds like they are connected to one another.